next-tavla/src/Board/scenarios/Table/components/TableHeader/styles.module.css

```diff
@@ -1,5 +1,7 @@
 .tableHeaderWrapper {
     display: flex;
+    flex-direction: row;
+    justify-content: space-between;
     align-items: center;
     min-height: 2em;
 }
```

Comment:
Hele denne klassen kan vel replikeres med hjelpeklassene våre. Du kan eventuelt lage hjelpeklassene for min-height